### PR TITLE
Update readme, license and pyproject

### PR DESCRIPTION
**Pull request by mateuszStolarski**
- Updated readme
- New line in license Copyright
- Author in pyproject.toml from cookiecutter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,52 @@ Simple template for a Python package.
...
+
+## Developer Tips
+
+### General
+
+Try to keep your code simple. \
+Remember to declaring types and try to avoid type `Any`. \
+Use apropriete commit names to make it easier for other people understanding your code (Convetional Commits specification - https://www.conventionalcommits.org/en/v1.0.0/)
+
+### Tests
```
Comment (adampirog) [inline README.md:26]: I suggest committing to `pytest` as our testing tool

File: cookiecutter.json [hunk 2]
```diff
@@ -3,5 +3,7 @@
   "package_name": "{{ cookiecutter.project_name|lower()|replace(' ', '_')|replace('-', '_')|trim() }}",
   "package_slug": "{{ cookiecutter.package_name|replace('_', '-') }}",
   "short_project_description": "",
+  "full_name": "",
+  "email": "",
```
Comment (adampirog) [inline cookiecutter.json]: I would drop the email

File: {{cookiecutter.package_slug}}/pyproject.toml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 [project]
 name = "{{ cookiecutter.package_name }}"
 authors = [
-    {name = "Adam Pirog"},
-    {email = "pirog.adam@gmail.com"},
+    {name = "{{cookiecutter.full_name}}"},
```
Comment (adampirog) [inline {{cookiecutter.package_slug}}/pyproject.toml]: Add if name empty -> skip authors section

File: {{cookiecutter.package_slug}}/pyproject.toml [hunk 4]
```diff
@@ -1,9 +1,12 @@
 [project]
 name = "{{ cookiecutter.package_name }}"
+{% if cookiecutter.authors != "" %}
 authors = [
-    {name = "Adam Pirog"},
-    {email = "pirog.adam@gmail.com"},
+    {% for author in cookiecutter.authors.split(',') %}
```
Comment (adampirog) [inline {{cookiecutter.package_slug}}/pyproject.toml]:
```suggestion
    {% for author in cookiecutter.authors.split(',') -%}
```
Authors get too spread out.

Really nice touch with the loop tho

File: {{cookiecutter.package_slug}}/pyproject.toml [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
+    {name = "{{ author.strip() }}"}{% if not loop.last %},{% endif %}
+    {% endfor %}
```
Comment (adampirog) [inline {{cookiecutter.package_slug}}/pyproject.toml]:
```suggestion
   {% endfor -%}
```
Same as above

File: cookiecutter.json [hunk 6]
```diff
@@ -3,5 +3,6 @@
   "package_name": "{{ cookiecutter.project_name|lower()|replace(' ', '_')|replace('-', '_')|trim() }}",
   "package_slug": "{{ cookiecutter.package_name|replace('_', '-') }}",
   "short_project_description": "",
+  "authors": "{{ cookiecutter.authors | default('') }}",
```
Comment (adampirog) [inline cookiecutter.json]: I'm pretty sure you can just leave it as `"authors": ""`, but this works as well

**Review by adampirog — APPROVED** [review]

**Review by pbrodka — APPROVED** [review]

**Pull request merged**